teuthology/suite/__init__.py

```diff
@@ -28,6 +28,7 @@ def process_args(args):
         'flavor': 'kernel_flavor',
         '<config_yaml>': 'base_yaml_paths',
         'filter': 'filter_in',
+        'no_validate_refs': 'validate_refs',
```

Comment:
for clarity we dont really have to rename the arg and avoid "value = not value" negation, the if branch could check use the negation check.

Reply:
Indeed, my thought was just that `if not no_validate_refs:` looks weird, so better to invert here, since there doesn't appear to be a convenient way to make an optional boolean parameter true by default.

Reply:
if no_validate_refs is False: can be used, and there is also else: if we not looks weird, double negation and changing the arg is confusing for code readability later on.